### PR TITLE
Rename and Attempts to add Type Input for Variables

### PR DESCRIPTION
**Pull request by WarpWing**
Don't commit this change YET, something is wrong pls halp

### CODE REVIEW

**Comment by WarpWing** [comment]
I fixed le issue. You can review and pull this request

File: Calculators/College/DebtCalculator.py [hunk 1]
```diff
@@ -1,12 +1,12 @@
-debt = 95000
-interest_percent = .059
-yearly_salary = None
-hourly_salary = 20
-debt_repayment_percent = .2
-debt_repayment_length = 0
-work_days = 261
-work_hours = 8
-interest = 0
+debt = int(input('Debt: '))
```
Comment (WarpWing) [inline Calculators/College/DebtCalculator.py:1]: I had to manually converted the variables from string to int. You think we could make this conversion more efficient or you want to leave it as it is?

**Review by WarpWing — COMMENTED** [review]
You can review the code :D

**Comment by EnderBro1000** [comment]
For this one, I wanted to input variables like I had it previously. Since only some variables will change some times, I don't want to type everything in every time when I only want to change a single variable

**Pull request closed without merging**